apps/src/aichat/views/ChatMessage.tsx

```diff
@@ -132,6 +136,13 @@ const ChatMessage: React.FunctionComponent<ChatMessageProps> = ({message}) => {
         displayModelUpdateMessage(message, () =>
           dispatch(removeModelUpdateMessage(message.id))
         )}
+
+      {isModelReset(message.role) &&
```

Comment:
I _think_ the new code here and the changes to `displayModelUpdateMessage` could be avoided if we update the following to include the "has been updated" suffix:

https://github.com/code-dot-org/code-dot-org/blob/ad1fa5624d940f6f3ca0d18aff73748e3f3ec0b8/apps/src/aichat/redux/aichatRedux.ts#L233

And treat this notification as a "model update", rather than a new role (if I squint, calling it a model update doesn't seem too far off to me)